modules/ROOT/pages/_partials/design-center.adoc

```diff
@@ -57,6 +58,9 @@ Supported fragments type are:
 * `user-documentation` 
 * `oas-components`
 * `json-schema`| `--type raml-fragment --fragmentType user-documentation`
+| `--version` |The project version | `--version 1.0.2`
+| `--format` | The project format. This flag is only available for OAS API's. +
```

Comment:
```suggestion
| `--format` | The project format. This flag is only available for OAS APIs. +
```

Reply:
(Globally)
The "The" article should not be included in the sentence. It should read "Project format" instead of "The project format".